src/skills-builder/skills-builder-steps/select-preferences/CareerInterestCategorizinator.jsx

```diff
@@ -39,24 +41,37 @@ const CareerInterestCategorizinator = () => {
     }
   };
 
+  // state for controlling every dropdown menu
+  const [dropdownState, setDropdownState] = useState(FORM_VALUES);
+
   const populateDropdown = (category, title = category) => {
     const currentCategory = careerList[category];
+    const controlState = dropdownState[category];
+
+    const handleSelectChange = (e) => {
+      const { value } = e.target;
+      setDropdownState({
+        ...FORM_VALUES,
+        [category]: value,
+      });
+      handleCareerInterestSelect(value);
+    };
 
     return (
-      <DropdownButton
-        id={category}
-        title={title}
-        variant="light"
+      <Form.Control
+        as="select"
+        value={controlState}
+        onChange={handleSelectChange}
+        className="select-width"
+        leadingElement={controlState && (<Icon src={Verified} />)}
+        floatingLabel={controlState && title}
       >
-        {currentCategory.map((job) => (
-          <Dropdown.Item
-            key={job.JobTitle}
-            onClick={() => handleCareerInterestSelect(job.JobTitle)}
-          >
-            {job.JobTitle}
-          </Dropdown.Item>
+        <option disabled={controlState}>{title}</option>
+        {currentCategory.map((job, idx) => (
+          // eslint-disable-next-line react/no-array-index-key
```

Comment:
Do we need this override since there IS a key?

Reply:
Yes the override is saying not to use `index` as the key because this is not recommended by React _if the order of the items is subject to change_: https://legacy.reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,an%20index%20as%20a%20key.

Since the order isn't changing, I believe it is safe to override the lint rule